extensions/images.go

```diff
@@ -164,7 +171,7 @@ func createImageFunctions() { //nolint:funlen // this is a group of related func
 		Name:       "image.new",
 		MinArgs:    3,
 		MaxArgs:    3,
-		Help:       "create a new RGBA image of the name and size, image starts entirely transparent",
+		Help:       "create a new NRGBA image of the name and size, image starts entirely transparent",
```

Comment:
no one reading this will understand NRGBA :grin:

Reply:
it's a search and replace, but the distinction is important, it means non pre multiplied ie you can have values of RGB higher than A while in RGBA you can't

Reply:
exactly, my point is the fact someone calling the help should get a better explanation than just an obscure acronym :grimacing:

Reply:
fixed!